### PR TITLE
Task dev 8

### PR DESCRIPTION
**Pull request by julialips**

### CODE REVIEW

File: Task2.Tests/UnitTest1.cs [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        [TestCase("сде+лать", "зд'элат'")]
+        [TestCase("я", "йа")]
+        [TestCase("я+", "йа")]
+        [TestCase("прст", "прст")]
+        [TestCase("аа", "аа")]
+        public void TestMethod1(string word, string expected)
+        {
+            var conv = new Converter();
+            var actual = conv.Convert(word);
+            Assert.AreEqual(expected,actual);
```
Comment (AlexeiRomankov) [inline Task2.Tests/UnitTest1.cs:21]: add space after ','

File: Task2.Tests/UnitTest1.cs [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        }
+
+        [TestCase("МОЛОКО+","МАЛАКО")]
+        [TestCase("мОлОКо+","мАлАКо")]
+        public void BigLetters(string word, string expected)
+        {
+            var conv = new Converter();
+            var actual = conv.Convert(word);
+            Assert.AreEqual(expected, actual);
```
Comment (AlexeiRomankov) [inline Task2.Tests/UnitTest1.cs:30]: it is better to add messages for assertions

File: Task2.Tests/UnitTest1.cs [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+            Assert.Throws<FormatException>
+            (
+                () => conv.Convert(word)
+            );
+        }
+
+        [TestCase(null)]
+        public void TestNullString(string word)
+        {
+            Assert.Throws<Exception>
```
Comment (AlexeiRomankov) [inline Task2.Tests/UnitTest1.cs:48]: it is better to use lower exceptions

**Comment by julialips** [comment]
Task_Dev-8

**Pull request merged**